telepict/templates/game.html

```diff
@@ -71,24 +71,7 @@
     <button type="button" onclick="passWriting()">Pass</button>
   </div>
   <div id="image-input-div" style="display: none">
-    <!-- http://www.williammalone.com/articles/create-html5-canvas-javascript-drawing-app/ -->
-    <select id="color-select">
-      <option value="#000000" selected>Black</option>
-      <option value="#df4b26">Red</option>
-      <option value="#cb3594">Purple</option>
-      <option value="#659b41">Green</option>
-      <option value="#ffcf33">Yellow</option>
-      <option value="#986928">Brown</option>
-    </select>
-    <select id="size-select">
-      <option value="1">Tiny</option>
-      <option value="3">Small</option>
-      <option value="5" selected>Normal</option>
-      <option value="8">Large</option>
-      <option value="14">Huge</option>
-    </select>
-    <canvas id="image-canvas" width="{{ config['CANVAS_WIDTH'] }}" height="{{ config['CANVAS_HEIGHT'] }}" style="border: 1px solid #000000;"></canvas>
-    <button type="button" onclick="clearCanvas()">Clear</button>
+    <div id="image-canvas" style="width: {{ config['CANVAS_WIDTH'] }}; height: {{ config['CANVAS_HEIGHT'] }}"></div>
```

Comment:
```suggestion
    <div id="image-canvas" style="width: {{ config['CANVAS_WIDTH'] }}px; height: {{ config['CANVAS_HEIGHT'] }}px"></div>
```

@dpitch40 it was intended to match sizes but apparently I forgot to specify a unit here. This should fix it.

Reply:
![CanvasDraw](https://user-images.githubusercontent.com/1520538/86521929-47221880-be1c-11ea-9855-77fe47783a6b.png)\

I see one on the top and presumably left edges, but not on the right or bottom.